CONTRIBUTING.md

```diff
@@ -229,3 +230,7 @@ Please run your code through:
     with a clear title and description against the `master` branch. All tests must be passing before we will review the PR.
 
 If you have any additional questions, please feel free to [email](mailto:dx@sendgrid.com) us or create an issue in this repo.
+
+<a name="code-reviews"></a>
+## Code Reviews
+If you can, please look at open PRs and review them. Give feedback and help us merge these PRs much faster! If you don't know how, Github has some great information on how to review a Pull Request.
```

Comment:
@jlax47 ,

I just noticed that the link from #525  is missing here.

* https://help.github.com/articles/about-pull-request-reviews/

Reply:
@delirious-lettuce,

Right you are. Sorry about that. I just updated it and pushed the changes into the same commit, it's showing up for me, can you confirm?